http/vibe/http/server.d

```diff
@@ -1733,7 +1733,7 @@ private void listenHTTPPlain(HTTPServerSettings settings, HTTPServerContext cont
 
 
 private void handleHTTPConnection(TCPConnection connection, HTTPListenInfo listen_info)
-@safe {
+@trusted {
```

Comment:
This one was actually left over from a change where the allocator was instantiated outside of the request handler loop (will remove). But as much as the need for `@trusted` is annoying in the general case, here it actually serves its purpose, as custom allocators are involved (`deallocateAll()` in particular). Would of course be nice to be able to implement `@safe` wrappers with DIP1000...